src/main/java/gift/entity/Member.java

```diff
@@ -0,0 +1,40 @@
+package gift.entity;
+
+public class Member {
+    private Long id;
+    private String email;
+    private String password;
+    private String role = "USER";
```

Comment:
```suggestion
public class Member {
    private final Long id;
    private final Email email;
    private final Password password;
    private final Role role;
```
- 객체를 불변(Immutable)하게 설계해보는건 어떨까요? [불변객체](https://www.brainbackdoor.com/95248237-c6a4-4489-a51e-6c548c0c103c)에 대해서도 학습해보시겠어요?
- 원시..값.......

Reply:
이메일과 비밀 번호는 추후 수정하는 기능을 도입할 수 있을 것이라고 생각하여 id와 role만 불변 처리 하였습니다!

원시값을 모두 포장하였습니다!